### ISSUE TITLE
Graph isn't declared in the current scope

### ISSUE THREAD

**Issue by Wiering** [goostengine/goost-examples#3]
Just downloaded Goost for the first time and trying out the examples.

**Which demo project or script is affected:**
goost-examples-gd3/2D/Geometry
goost-examples-gd3/core/Graph

**OS/device including version:**
Windows 10, Goost v1.2 stable

triangulate_delaunay.gd error(5,1) / mst.gd error(8,1)
var graph = Graph.new()
Parser Error: The identifier "Graph" isn't declared in the current scope.

**Comment by Xrayez** [comment]
Yeah, unfortunately the latest Goost 1.2 stable release doesn't contain the new `Graph` class. The next release (Goost 2.0) was supposed to have this class, but I no longer maintain Goost myself, nor using Godot.

However, if you're motivated enough, you can build Goost from source yourself, see documentation: https://goost.readthedocs.io/en/latest/usage/building.html

Or you can simply ignore the `Graph` class if you don't actually need it, of course. You can download a previous version of `goost-examples` which doesn't contain the latest `Goost` features, here: https://github.com/goostengine/goost-examples/tree/c027a66c14710c8246b72df9f08f7e90f3cc702f
